index.html

```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+        <ul class="nav-buttons box-nav">
+          <li><a href="">HTML</a></li>
+          <li><a href="">Bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
         </ul>
-      </nav>  
+
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div>   
+
+      <div class="works-box">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+        <ul class="nav-buttons box-nav">
+          <li><a href="">HTML</a></li>
+          <li><a href="">Bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div> 
+
+      <div class="works-box">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+        <ul class="nav-buttons box-nav">
+          <li><a href="">HTML</a></li>
+          <li><a href="">Bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div> 
+
+      <div class="works-box">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+        <ul class="nav-buttons box-nav">
+          <li><a href="">HTML</a></li>
+          <li><a href="">Bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div> 
+
+      <div class="works-box">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+        <ul class="nav-buttons box-nav">
+          <li><a href="">HTML</a></li>
+          <li><a href="">Bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div> 
+
+      <div class="works-box">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+        <ul class="nav-buttons box-nav">
+          <li><a href="">HTML</a></li>
+          <li><a href="">Bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+
+        <button class="btn-type-1 box-btn">See Project</button>
+      </div> 
+
+
+
+
+
     </section>
 
+    <section id="about-me">
+      <div id="resume">
+        <h1>About me</h1>
```

Comment:
- [ ] Kindly use only one h1 tag in your web application. It is best SEO practice to use a single h1 tag per web page. Kindly change this to another heading tag `h2 - h5`.